week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -72,9 +78,13 @@ function getEligibleStudents() {
   Hint: To complete the function, search how to change text to lower or upper case by using string method.
 */
 
-function getEligibleStudents2() {
-}
+function getEligibleStudents2(names) {
+  let findWithA = names.filter(
+    (newArray) => newArray[0].startsWith("A") || newArray[0].startsWith("a")
+  );
```

Comment:
Here the function is correct but you haven't followed the instruction. Fixed it please.